pkg/gitreceive/k8s_util.go

```diff
@@ -25,6 +24,11 @@ const (
 	dockerSocketPath = "/var/run/docker.sock"
 )
 
+var (
+	slugBuilderImage   = getenv("SLUGBUILDER_IMAGE_NAME", "quay.io/deisci/slugbuilder:v2-beta")
+	dockerBuilderImage = getenv("DOCKERBUILDER_IMAGE_NAME", "quay.io/deisci/dockerbuilder:v2-beta")
```

Comment:
@kmala just went over it. only https://github.com/deis/builder/pull/243#discussion_r55615726 remains. thanks for making the changes